alpine-common/src/main/java/alpine/Config.java

```diff
@@ -101,6 +101,7 @@ public enum AlpineKey implements Key {
         WORKER_THREAD_MULTIPLIER               ("alpine.worker.thread.multiplier",   4),
         DATA_DIRECTORY                         ("alpine.data.directory",             "~/.alpine"),
         SECRET_KEY_PATH                        ("alpine.secret.key.path",            null),
+        SECRET_API_KEYPREFIX                   ("alpine.secret.api.keyprefix",       "dta_"),
```

Comment:
While DT is the main adopter of Alpine, it's technically not the only app using it. So the default prefix should not be specific to DT (`dta`). I'd suggest either leaving it empty or using a generic `alpine` prefix. This propertly will need to be overridden in DT's `application.properties`.